swagger_spec_validator/validator20.py

```diff
@@ -499,6 +511,14 @@ def validate_definition(definition, deref, def_name=None, visited_definitions_id
                 visited_definitions_ids=visited_definitions_ids,
             )
 
+    if 'additionalProperties' in definition:
+        validate_definition(
+            definition=definition.get('additionalProperties'),
+            deref=deref,
+            def_name='{}/{}'.format(def_name, 'additionalProperties'),
```

Comment:
Why have `additionalProperties` as an argument here? You could put it into the string directly, like you did for `items`.

Reply:
yep good point.